source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,245 @@
+.. _realm-sync-migrate-modes:
+
+=========================
+Migrate Device Sync Modes
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+Atlas App Services Apps can have one of two Device Sync modes: Partition-Based Sync
+and Flexible Sync. Partition-Based Sync is an older mode and you should consider
+migrating to Flexible Sync.
+
+Migrating an App that uses Partition-Based Sync to Flexible Sync is an automatic
+process. Other than upgrading the SDK version, migrating sync modes doesn't require 
+any changes to your client app code. Your App's backend will automatically translate
+the client's Partition-Based Sync requests into Flexible Sync.
+
+Migrating results in Partition-Based Sync clients communicating with a
+Flexible Sync Atlas App Services App.
+
+Requirements
+------------
+
+Your App Services App:
+
+- Must currently use Partition-Based Sync
+
+Linked Atlas Cluster:
+
+- Must be running mongodb 5.0+ for Flexible Sync compatibility
+- Must have storage auto-scaling enabled
+- Must be M10+ to access this setting
+- Must have time-based oplog setting enabled
+
+Client Apps:
+
+- ``realm-core`` vTBD+, which means these SDK versions: TBD
+
+.. _realm-sync-migrate-what-to-know:
+
+What to Know Before migrating
+-----------------------------
+
+Migrating your App's Sync mode involves changes that affect both client apps and
+your App's backend. You should be aware of the side effects and plan ahead.
+
+How long a migration takes is directly proportional to the amount of data that
+needs to be migrated. The more data, the longer it will take to migrate.
+
+Migrations are not permenant immediately after completion. They can be canceled
+during the migration or rolled back within a certain timeframe after the migration
+is finished.
+
+Progress notifications will not work after migration because Flexible Sync does
+not yet support them.
+
+Client apps that use an SDK with with a realm-core version lower than TBD will
+experience a client reset after migration. You should use a 
+:ref:`client reset <handle-a-client-reset>` handler with recovery enabled.
+
+.. _realm-sync-migrate-stages:
+
+Migration Stages
+~~~~~~~~~~~~~~~~
+
+There are three stages to a Sync mode migration. You can see your migration's
+stage in the App Services UI.
+
+#. Disabling Partition-Based Sync
+   
+   At this stage, client apps can connect to your App's backend, but local writes 
+   will not sync. These writes will be lost if you don't have a 
+   :ref:`client reset handler <handle-a-client-reset>` to recover unsynced changes.
+	 
+   Clients can also receive updates to MongoDB data, but they won't
```

Comment:
@kmorkos, "MongoDB data" in this instance refers to the linked cluster, correct?

Reply:
correct